README.md

```diff
@@ -70,6 +70,20 @@ public class DirectoryWatchingUtility {
 }
 ```
 
+## Configuration
+
+By default, DirectoryWatcher will try to prevent duplicate events (e.g. Windows will emit duplicate modify events when file is changed). This is done by creating hash for every file encountered and keeping that hash in memory.
+This might result in slower performance, because library has to calculate the hash of the entire file. In addition, some events are not emited, because file was locked by another process when library tried to calculate the hash. 
```

Comment:
Same thing: "*the* library", "*the* file".

I think it would be better to state the second sentence as: "In addition, some events may not be emitted if DirectoryWatcher encounters a file that is locked by another process while computing the hash."